stack_data/formatting.py

```diff
@@ -189,9 +191,19 @@ def format_line(self, line: Line) -> str:
                         + self.executing_node_underline * (end - start)
                         + "\n"
                 )
-
         return result
 
+
+    def format_blank_lines(self, blank_line):
+        result = ""
+        if self.current_line_indicator:
+            result = " " * len(self.current_line_indicator)
+        result = result or "   "
```

Comment:
```suggestion
        if self.current_line_indicator:
            result = " " * len(self.current_line_indicator)
        else:
            result = "   "
```

This seems equivalent to me.